internal/app.go

```diff
@@ -49,7 +49,10 @@ func (a *App) Run(ctx context.Context) error {
 		slogger := logger.New(*agentConfig.Log)
 		slog.SetDefault(slogger)
 
-		slog.InfoContext(ctx, "Starting NGINX Agent")
+		slog.InfoContext(ctx, "Starting NGINX Agent",
+			slog.String("version", a.version),
```

Comment:
We have being using loosely typed fields for other log statements.
We probably should be consistent in the way we log things.